web/client/utils/ConfigUtils.js

```diff
@@ -169,6 +169,10 @@ export const setApiKeys = function(layer) {
     if (layer.type === 'mapquest') {
         layer.apiKey = defaultConfig.mapquestApiKey;
     }
+    if (layer.type === 'tileprovider' && ['MapBoxStyle', 'MapBox'].includes(layer.provider)) {
+        // include an empty string if missing to avoid errors in the layer url template
+        layer.accessToken = defaultConfig.mapboxAccessToken || '';
```

Comment:
what if you include this in the defaultConfig object as a default value?

similar to what is done with mapquestApiKey, so you can skip || '' I suppose

Reply:
thanks @MV88 , I'll check it and replace